hack/update-template-oraclelinux.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+# Functions in this script assume error handling with 'set -e'.
+# To ensure 'set -e' works correctly:
+# - Use 'set +e' before assignments and '$(set -e; <function>)' to capture output without exiting on errors.
+# - Avoid calling functions directly in conditions to prevent disabling 'set -e'.
+# - Use 'shopt -s inherit_errexit' (Bash 4.4+) to avoid repeated 'set -e' in all '$(...)'.
+shopt -s inherit_errexit || error_exit "inherit_errexit not supported. Please use bash 4.4 or later."
+
+function oraclelinux_print_help() {
+	cat <<HELP
+$(basename "${BASH_SOURCE[0]}"): Update the Oracle Linux image location in the specified templates
+
+Usage:
+  $(basename "${BASH_SOURCE[0]}") [--version <version>] <template.yaml>...
+
+Description:
+  This script updates the Oracle Linux image location in the specified templates.
+  Image location basename format:
+
+	OL<version>U<release>_<arch>-kvm[-cloud]-b<build number>.qcow2
+
+  Published Oracle Linux image information is fetched from the following URLs:
+
+    OL8:
+	  x86_64: https://yum.oracle.com/templates/OracleLinux/ol8-template.json
+	  aarch64: https://yum.oracle.com/templates/OracleLinux/ol8_aarch64-cloud-template.json
+	  
+	OL9:
+	  x86_64: https://yum.oracle.com/templates/OracleLinux/ol9-template.json
+	  aarch64: https://yum.oracle.com/templates/OracleLinux/ol9_aarch64-cloud-template.json
+
+  The downloaded files will be cached in the Lima cache directory.
+
+Examples:
+  Update the Oracle Linux image location in templates/**.yaml:
+  $ $(basename "${BASH_SOURCE[0]}") templates/**.yaml
+
+  Update the Oracle Linux image location to version 9 in ~/.lima/oraclelinux/lima.yaml:
+  $ $(basename "${BASH_SOURCE[0]}") --version 9 ~/.lima/oraclelinux/lima.yaml
+  $ limactl factory-reset oraclelinux
+
+Flags:
+  --version <version>  Use the specified Oracle Linux <version>.
```

Comment:
Shouldn't this be `--version-major`?
For consistency with alma and rocky

Reply:
As far as I’ve found, Oracle Linux does not have major or minor versions, but instead uses descriptions like “version 8 update 10,” so I’ve aligned with that format.

Reply:
Their "version" number equates to the major version of RHEL/Alma/Rocky, and their "update" number equates to the minor version.

Reply:
They also call it "8.10": https://docs.oracle.com/en/operating-systems/oracle-linux/8/relnotes8.10/

> Release Notes for Oracle Linux 8.10

Reply:
reworded `version` to `major version`, `release` to `minor version`